src/auto/dbus_method_invocation.rs

```diff
@@ -0,0 +1,168 @@
+                self.to_glib_none().0,
+            ))
+        }
+    }
+
+    pub fn get_object_path(&self) -> Option<GString> {
+        unsafe {
+            from_glib_none(gio_sys::g_dbus_method_invocation_get_object_path(
+                self.to_glib_none().0,
+            ))
+        }
+    }
+
+    pub fn get_parameters(&self) -> Option<glib::Variant> {
+        unsafe {
+            from_glib_none(gio_sys::g_dbus_method_invocation_get_parameters(
+                self.to_glib_none().0,
+            ))
+        }
+    }
+
+    pub fn get_property_info(&self) -> Option<DBusPropertyInfo> {
+        unsafe {
+            from_glib_none(gio_sys::g_dbus_method_invocation_get_property_info(
+                self.to_glib_none().0,
+            ))
+        }
+    }
+
+    pub fn get_sender(&self) -> Option<GString> {
+        unsafe {
+            from_glib_none(gio_sys::g_dbus_method_invocation_get_sender(
+                self.to_glib_none().0,
+            ))
+        }
+    }
+
+    //pub fn get_user_data(&self) -> /*Unimplemented*/Option<Fundamental: Pointer> {
+    //    unsafe { TODO: call gio_sys:g_dbus_method_invocation_get_user_data() }
+    //}
+
+    pub fn return_dbus_error(&self, error_name: &str, error_message: &str) {
+        unsafe {
+            gio_sys::g_dbus_method_invocation_return_dbus_error(
+                self.to_glib_full(),
+                error_name.to_glib_none().0,
+                error_message.to_glib_none().0,
+            );
+        }
+    }
+
+    //pub fn return_error(&self, domain: glib::Quark, code: i32, format: &str, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs) {
+    //    unsafe { TODO: call gio_sys:g_dbus_method_invocation_return_error() }
+    //}
+
+    pub fn return_error_literal(&self, domain: glib::Quark, code: i32, message: &str) {
+        unsafe {
+            gio_sys::g_dbus_method_invocation_return_error_literal(
+                self.to_glib_full(),
+                domain.to_glib(),
+                code,
+                message.to_glib_none().0,
+            );
+        }
+    }
+
+    //pub fn return_error_valist(&self, domain: glib::Quark, code: i32, format: &str, var_args: /*Unknown conversion*//*Unimplemented*/Unsupported) {
+    //    unsafe { TODO: call gio_sys:g_dbus_method_invocation_return_error_valist() }
+    //}
+
+    pub fn return_gerror(&self, error: &glib::Error) {
+        unsafe {
+            gio_sys::g_dbus_method_invocation_return_gerror(
+                self.to_glib_full(),
+                error.to_glib_none().0,
+            );
+        }
+    }
+
+    pub fn return_value(&self, parameters: Option<&glib::Variant>) {
+        unsafe {
+            gio_sys::g_dbus_method_invocation_return_value(
+                self.to_glib_full(),
+                parameters.to_glib_none().0,
+            );
+        }
+    }
+
+    pub fn return_value_with_unix_fd_list<P: IsA<UnixFDList>>(
```

Comment:
Need `cfg_condition`

Reply:
yep, I just fixed that in my local copy, after noticing the Appveyor failure